components/bio/bio.scss

```diff
@@ -13,6 +13,12 @@
     }
   }
 
+  @media screen and (max-width: 500px) {
+    .social-media span {
+      font-size: 40px;
```

Comment:
From https://github.com/mozilla/network-pulse/issues/805#issuecomment-343208753 and https://github.com/mozilla/network-pulse/issues/805#issuecomment-349737177
> Icon size is ok, just made the hint area bigger please.

> @mmmavis could you please make the profile icons (Twitter, LinkedIn, and GitHub) the same height as the post icons are (heart and Twitter share). And then make all their hit areas 40x40px.

We want to make the hint areas bigger while keeping the icon size (font size in this case) the same. So increasing the font-size to 40px is not what we want there 😬 .

Reply:
@mmmavis , sure I'll make the changes

Reply:
Could you explain hit area ? Do you mean padding ? And do I have to change the hit area to 40px in mobile view or normal view.